docs/recent_activity/recent_activity.md

```diff
@@ -0,0 +1,54 @@
+---
+description: Monitor recent activity logged actions.
+---
+
+# Recent activity log
+
+## Activity list
+
+On the Back Office, in Admin -> Activity list, you can consult what happens recently on the database.
```

Comment:
It's not accurate and may give false impression that we're tracking database changes.

```suggestion
On the Back Office, in Admin -> Activity list, you can consult what actions were taken by users recently.
```